### PR TITLE
Feat/project details

### LINKED ISSUE #203: As a CIF user, I want to view a project details, so that I can, at a glance, view high level information about the project
#### Description:
As a CIF user, I want to view a project details, so that I can, at a glance, view high level information about the project.

#### Acceptance Criteria:
Given that I navigate to the `/cif/project/<projectRelayId>` page
When the page loads
Then I can see a project's information

#### Development Checklist:
- [ ] The project page loads data from the `project`, `project_contact`, `project_manager`, etc, tables, instead of the `form_change` table
- [ ] Design should remain minimalist (e.g. see https://design-system.service.gov.uk/components/summary-list/) and will be revised after v1
- [ ] Meets the DOD

#### Notes:
- UX questions: do we want a single page or tabs? This could be updated easily if we make components for the different sections

Personas
[Program Development Team](https://bcgov.sharepoint.com/:b:/t/00608-CIFInternalUsers/EcVHGc4X8mhIvQHGSaGJmyAB9F1iBe7MSH1Y1fBKUc1skA?e=fzATIc)

[Operations Team](https://bcgov.sharepoint.com/:b:/t/00608-CIFInternalUsers/EaswKcprmuNKpEuY29a-Os0BG-h5u-Ztou7UuzK3x0xfjw?e=e8oR1i)

[Tech Team ](https://bcgov.sharepoint.com/:b:/t/00608-CIFInternalUsers/Ec2AmUecYCdDlmURctGwTbMBUMGZXlUyyt5HQZJdpjeejA?e=0axXlm)

### PR DESCRIPTION
**Pull request by BCerki**
Resolves #203

Allows viewing a project's details. Code is based on `cas-cif/app/pages/cif/contact/[contact].tsx` and unit test on `app/tests/unit/pages/contact/[contact].test.tsx`. I didn't do an e2e test since the contact display doesn't have one, but happy to add one if needed.

### CODE REVIEW

File: app/pages/cif/project/[project]/index.tsx [hunk 1]
```diff
@@ -4,24 +4,140 @@ import { graphql, usePreloadedQuery } from "react-relay/hooks";
 import { ProjectOverwiewQuery } from "__generated__/ProjectOverwiewQuery.graphql";
 import withRelayOptions from "lib/relay/withRelayOptions";
 
-const pageQuery = graphql`
+export const pageQuery = graphql`
   query ProjectOverwiewQuery($project: ID!) {
```
Comment (BCerki) [inline app/pages/cif/project/[project]/index.tsx:8]: I was going to fix the typo in `ProjectOverwiewQuery` but then it occured to me changing a query name might cause problems elsewhere?

File: app/pages/cif/project/[project]/index.tsx [hunk 2]
```diff
@@ -4,24 +4,140 @@ import { graphql, usePreloadedQuery } from "react-relay/hooks";
 import { ProjectOverwiewQuery } from "__generated__/ProjectOverwiewQuery.graphql";
 import withRelayOptions from "lib/relay/withRelayOptions";
 
-const pageQuery = graphql`
+export const pageQuery = graphql`
```
Comment (BCerki) [inline app/pages/cif/project/[project]/index.tsx:7]: I was going to change the variable name to `projectOverviewQuery`, but then it occurred to me that a name change might cause problems elsewhere?

File: app/pages/cif/project/[project]/index.tsx [hunk 3]
```diff
@@ -1,29 +1,145 @@
...
+        }
+      }
+      projectManagersByProjectId {
+        edges {
+          node {
+            cifUserByCifUserId {
+              firstName
+              lastName
+              id
+            }
```
Comment (BCerki) [inline app/pages/cif/project/[project]/index.tsx:47]: I still don't totally follow why I sometimes have to go through edges and nodes but sometimes I just go straight to the field I'm looking for

File: app/pages/cif/project/[project]/index.tsx [hunk 4]
```diff
@@ -1,29 +1,145 @@
...
+        <dt>Funding Stream</dt>
+        <dd>
+          {
+            project.fundingStreamRfpByFundingStreamRfpId
+              .fundingStreamByFundingStreamId.description
+          }
+        </dd>
+
+        <dt>Funding Stream RFP</dt>
+        <dd>{project.fundingStreamRfpByFundingStreamRfpId.year}</dd>
```
Comment (BCerki) [inline app/pages/cif/project/[project]/index.tsx:95]: Have I got the correct data here? On the form it looked like the dropdown for Funding Stream RFP is just a list of years

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]
Looking good! A couple of tweaks that can be done.

The spurious happo diffs are an issue, but not for this PR. We have a tech debt issue to solve those (#275 )

**Review by matthieu-foucault — APPROVED** [review]

**Review by matthieu-foucault — APPROVED** [review]

**Pull request merged**